tests/Agent/IntegrationTests/UnboundedIntegrationTests/Redis/StackExchangeRedisTests.cs

```diff
@@ -11,50 +10,56 @@
 using System.Linq;
 using Xunit;
 using Xunit.Abstractions;
+using MultiFunctionApplicationHelpers;
 
 namespace NewRelic.Agent.UnboundedIntegrationTests.Redis
 {
-    [NetFrameworkTest]
-    public class StackExchangeRedisTests : NewRelicIntegrationTest<RemoteServiceFixtures.BasicMvcApplication>
+    public abstract class StackExchangeRedisTestsBase<TFixture> : NewRelicIntegrationTest<TFixture> where TFixture : ConsoleDynamicMethodFixture
     {
-        private readonly RemoteServiceFixtures.BasicMvcApplication _fixture;
+        private readonly ConsoleDynamicMethodFixture _fixture;
 
-        public StackExchangeRedisTests(RemoteServiceFixtures.BasicMvcApplication fixture, ITestOutputHelper output)  : base(fixture)
+        public StackExchangeRedisTestsBase(TFixture fixture, ITestOutputHelper output) : base(fixture)
         {
             _fixture = fixture;
             _fixture.TestLogger = output;
+
+            _fixture.AddCommand($"StackExchangeRedisExerciser DoSomeWork");
+            _fixture.AddCommand($"StackExchangeRedisExerciser DoSomeWorkAsync");
+
             _fixture.Actions
```

Comment:
You could call `_fixture.AddActions` here and include your wait for log lines logic.

Reply:
Oh I see what you mean. I agree, that's a little cleaner. Done.